{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "77b1a574_5274ede9",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1000948
      },
      "writtenOn": "2024-01-30T23:09:17Z",
      "side": 1,
      "message": "recheck\n\ngetVolthaCode updated with debug statements to help figure out where the exception originated.",
      "revId": "253fa01becd7dbb5725e6aa7c7cf933aedfd9072",
      "serverId": "2a2bfe1b-c5c2-48ed-9ac8-16438ab24388"
    }
  ]
}